.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+on: push
+
+jobs:
+  test:
+    container:
+      image: haskell:${{ matrix.ghc }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ghc: ['8.10', '8.8']
```

Comment:
Add 8.6 too and please update tested-with in the cabal file accordingly.

Reply:
The haskell docker image seems to only support the last two major versions :( https://hub.docker.com/_/haskell

Reply:
You can use actions/setup-haskell instead.

Reply:
Done